### PR TITLE
Add Artemis server dependency

### PR DESCRIPTION
**Pull request by gogit**
Add Artemis server dependency

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,13 @@ on [Keep a CHANGELOG](http://keepachangelog.com/). This project adheres to
 
 ## [Un-released]
 
+### Added changed removed and sorted
```
Comment (purple52) [inline CHANGELOG.md]: These change log lines should be under separate headings (eg "Added", "Changed" and "Fixed") - see the http://keepachangelog.com for guidance on headings.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,13 @@ on [Keep a CHANGELOG](http://keepachangelog.com/). This project adheres to
...
+
+- Added jboss-logging slf4j-jboss-logging and jboss-logmanager libraries
```
Comment (purple52) [inline CHANGELOG.md]: I don't see any changes to the dependency management section, so where are the versions enforced?

Comment (gogit) [inline CHANGELOG.md]: Added the dependencies to the section.

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,21 @@ on [Keep a CHANGELOG](http://keepachangelog.com/). This project adheres to
 
 ## [Un-released]
 
+## [1.3.0] - 2016-09-26
```
Comment (purple52) [inline CHANGELOG.md]: All these changes should be under the heading [Unreleased] until we do a release. We're currently on1.2.0-SNAPSHOT, so the next release will be 1.2.0. But we don't update this file until we create the release tag.

Comment (gogit) [inline CHANGELOG.md]: made the changes

**Comment by gogit** [comment]
Branch is obsolete

**Pull request closed without merging**